packages/toolpad-core/src/AppProvider/LocalizationProvider.tsx

```diff
@@ -105,7 +103,47 @@ LocalizationProvider.propTypes /* remove-proptypes */ = {
   /**
    * Locale for components texts
    */
-  localeText: PropTypes.object,
```

Comment:
I believe this was deliberate

Reply:
Yes, but I felt that having the explicit props is helpful at least on the `LocalizationProvider` docs, it is typed as `object` for every other component. We can revert it if you think it doesn't make sense

Reply:
> We can revert it if you think it doesn't make sense

It makes sense, but I think it makes more sense to be consistent with X